src/tools/generate/generate_sitemap.js

```diff
@@ -78,7 +81,7 @@ const set_min_date = (date) => {
     if (date < min_publish_date) {
       return min_publish_date.toISOString().substr(0, 10);
     } else {
-      return date;
+      return date.toISOString().substr(0, 10);
```

Comment:
Just to confirm, last modified can be a date and not necessarily datetime?

Reply:
It has to be a date: https://www.sitemaps.org/protocol.html#lastmoddef

Reply:
Though I've only ever seen dates in the wild. E.g. https://web.dev/sitemap.xml